src/ObjCRuntime/Runtime.cs

```diff
@@ -49,6 +49,8 @@ public partial class Runtime {
 		static object lock_obj;
 		static IntPtr NSObjectClass;
 		static bool initialized;
+		static bool useCFNetworkHandler = false;
+		static bool useNSUrlSessionHandler = false;
```

Comment:
The assignment to the default value is unnecessary, except that the default is to use NSUrlSessionHandler:

```suggestion
		static bool useCFNetworkHandler;
		static bool useNSUrlSessionHandler = true;
```